src/kernels/installer/pyenv.node.ts

```diff
@@ -1,232 +0,0 @@
-import * as path from '../../platform/vscode-path/path';
```

Comment:
This seems like a bug. We should support pyenv. Maybe I forgot to port something from the python extension.

Reply:
@rchiodo this is the code we moved from Python into Jupyter for Python Installer. The files included unit tests and code checking pyenv but this is not used by us (Jupyter). We have our own pyenv support in https://github.com/microsoft/vscode-jupyter/blob/ecc0aaf4fab122f24375fe0e1afd9c59548dddca/src/kernels/installer/pyenv.node.ts#L1